server/sonar-web/src/main/js/components/SourceViewer/SourceViewerBase.tsx

```diff
@@ -300,29 +272,34 @@ export default class SourceViewerBase extends React.PureComponent {
       );
     };
 
-    this.props
-      .loadComponent(this.props.component, this.props.branch)
-      .then(onResolve, onFailLoadComponent);
+    this.props.loadComponent!(this.props.component, this.props.branch).then(
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Forbidden non null assertion ([typescript:S2966](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS2966))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fpopups&issues=AWHb6qYww6XGHKvPuQTE&open=AWHb6qYww6XGHKvPuQTE)

[ISSUE_KEY]: AWHb6qYww6XGHKvPuQTE